WalletWasabi.Fluent/Controls/Spectrum/SpectrumControl.cs

```diff
@@ -55,6 +55,18 @@ public SpectrumControl()
 				new GradientStop { Color = Color.Parse("#FF000D21"), Offset = 1 }
 			}
 		};
+
+		DispatcherTimer.Run(
+			() =>
+			{
+				if (IsVisible)
+				{
+					InvalidateVisual();
+				}
+				return true;
+			},
+			TimeSpan.FromMilliseconds((float)1000/60),
+			DispatcherPriority.Render);
```

Comment:
Would it make sense to not run it all the time, but when the control is visible?

```
renderTimer = new DispatcherTimer(DispatcherPriority.Render) { Interval = TimeSpan.FromMilliseconds((float)1000 / 60) };
renderTimer.Tick += (_, _) => InvalidateVisual();
```

And call start when IsVisible changed to true and call stop when changed to false?